job-service-container/pom.xml

```diff
@@ -420,6 +427,15 @@
                                                 <unpack>true</unpack>
```

Comment:
Just above the `assembly` node I can still see the `healthCheck` node and `mode` is still set to `none`.